src/functions-and-arrays.js

```diff
@@ -1,36 +1,113 @@
 // Iteration #1: Find the maximum
-
+function maxOfTwoNumbers(num1, num2){
+  if (num1 > num2){
+    return num1;
+  } else if (num1 < num2){
+    return num2;
+  } else if (num1 == num2){
+    return num1;
+  }
+}
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
-
+function findLongestWord(array){
+  if (array.length === 0) {
+    return null;
+  }
+  let word = "";
+  for(let i=0; i < array.length; i++) {
+    if (word.length < array[i].length) {
+      word = array[i];    
+    } if (array.length <= 0) {
+      callback(err , []);
+    }
```

Comment:
these lines are not necessary, as you have already checked if the array is empty on line 14.